chromium_src/components/payments/content/payment_request.h

```diff
@@ -0,0 +1,44 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_COMPONENTS_PAYMENTS_CONTENT_PAYMENT_REQUEST_H_
+#define BRAVE_CHROMIUM_SRC_COMPONENTS_PAYMENTS_CONTENT_PAYMENT_REQUEST_H_
+
+#include "brave/vendor/bat-native-ledger/include/bat/ledger/mojom_structs.h"
+#include "components/payments/content/initialization_task.h"
+#include "components/payments/content/payment_handler_host.h"
+#include "components/payments/content/payment_request_display_manager.h"
+#include "components/payments/content/payment_request_spec.h"
+#include "components/payments/content/payment_request_state.h"
+#include "components/payments/content/service_worker_payment_app.h"
+#include "third_party/blink/public/mojom/payments/payment_request.mojom.h"
+
+using payments::mojom::PaymentErrorReason;
+
+#if !defined(OS_ANDROID)
```

Comment:
if you were keeping the patch (which I don't think you need to) this would break on android. You need to define it to be nothing on android since you are using it in the original payment_request.h